get/manager.go

```diff
@@ -28,7 +28,7 @@ func GetManager(remoteBackend backend.Backend) error {
 	if viper.IsSet("cluster_manager") {
 		selectedClusterManager = viper.GetString("cluster_manager")
 	} else if nonInteractiveMode {
-		return errors.New("cluster_manager must be specified")
+		return errors.New("Cluster manager must be specified")
```

Comment:
This was purposefully named `cluster_manager` since in non interactive mode the configuration file must contain `cluster_manager`